src/tokens/TokenBalanceItem.tsx

```diff
@@ -28,6 +30,8 @@ export const TokenBalanceItem = ({
 }: Props) => {
   const { t } = useTranslation()
 
+  const hideWalletBalance = useSelector(hideWalletBalancesSelector)
```

Comment:
This would end up hiding the token balance everywhere this used, e.g., send/swap/cico token bottom sheets, old Assets screen, etc. We only want to hide this on the wallet tab.

Reply:
oof thanks for that!